File: app/src/main/java/com/example/myfirstapplication/MainActivity.java
package com.example.myfirstapplication;

import android.annotation.SuppressLint;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    private static final String TAG = "MainActivity";

    TextView textView;
    Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = findViewById(R.id.textView);
        button = findViewById(R.id.button);

        button.setOnClickListener(view -> updateText());
    }

    @SuppressLint("SetTextI18n")
    public void updateText() {
        textView.setText("Button clicked!");
        Log.d(TAG, "Button clicked!");
    }
}

File: Gallery/MainActivity.cs
using Android.App;
using Android.OS;
using Android.Runtime;
using Android.Widget;
using AndroidX.AppCompat.App;

namespace Gallery
{
    [Activity(Label = "@string/app_name", Theme = "@style/AppTheme", MainLauncher = true)]
    public class MainActivity : AppCompatActivity
    {

        Button GalleryB;
        Button WebViewB;
        Button PaintB;
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);
            Xamarin.Essentials.Platform.Init(this, savedInstanceState);
            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.activity_main);

            GalleryB = FindViewById<Button>(Resource.Id.GalleryBUT);
            WebViewB = FindViewById<Button>(Resource.Id.WebViewBUT);
            PaintB = FindViewById<Button>(Resource.Id.PaintBUT);


            GalleryB.Click += GalleryB_Click;
            WebViewB.Click += WebViewB_Click;
            PaintB.Click += PaintB_Click;
        }

        private void PaintB_Click(object sender, System.EventArgs e)
        {
            StartActivity(typeof(ActivityPaint));
        }

        private void GalleryB_Click(object sender, System.EventArgs e)
        {
            StartActivity(typeof(ActivityGallery));
        }

        private void WebViewB_Click(object sender, System.EventArgs e)
        {
            StartActivity(typeof(ActivityWebView));
        }

        public override void OnRequestPermissionsResult(int requestCode, string[] permissions, [GeneratedEnum] Android.Content.PM.Permission[] grantResults)
        {
            Xamarin.Essentials.Platform.OnRequestPermissionsResult(requestCode, permissions, grantResults);

            base.OnRequestPermissionsResult(requestCode, permissions, grantResults);
        }
    }
}